Textfield/Textfield.jsx

```diff
@@ -83,9 +91,9 @@ class TextfieldInput extends MaterialComponent {
           />
         )}
         {props.label &&
-        this.state.showFloatingLabel && (
-          <Label for={props.id}>{props.label}</Label>
-        )}
+          this.state.showFloatingLabel && (
+            <Label for={props.id}>{props.label}</Label>
+          )}
```

Comment:
I assume this is breaking ```prettier```.
Can you run ```npm run prettier:write``` and push?

Reply:
`npm run prettier:write` did not result in any committable changes

Reply:
Why did this change anyway? Was it ```prettier``` or did you change this by hand?

Reply:
Prettier is changing this. Should I turn off Prettier and push the old version?

Reply:
For debugging purposes:
- what ```prettier``` version do you have?
- what OS are you on?

Don't know yet why it wants to change it on your computer but not on Travis.

Reply:
I am using the Prettier VS Code plugin, v: 0.24.0 on MacOS Sierra (10.12.6)

Reply:
Haven't tried the plugin yet, maybe that's the issue?

Reply:
We actually don't have a ```prettier``` config file. The VS Code  plugin is looking for those :
- package.json - prettier key
- .prettierrc
- .prettier.config.js

Reply:
Hmm. How do we move forward?

Reply:
I guess you could still try disabling the plugin or running ```npm run prettier:write``` with VS Code closed, and if that works we might need to add a config file.

Reply:
I ran `npm i` to install Prettier locally in the project. Saved it and it fixed the formatting discrepancy.